spec/requests/oauth2_spec.rb

```diff
@@ -2,10 +2,38 @@
   let!(:login) { create(:login) }
 
   describe 'POST /token' do
+    before(:all) do
+      @orig_force_ssl = RailsApiAuth.force_ssl
+      RailsApiAuth.force_ssl = false
+    end
```

Comment:
Do the tests actually fail with the default value of `true`? I wonder whether it would be possible to fake SSL in RSpec tests.